website/docs/docs/getting-started/1-installation.md

```diff
@@ -0,0 +1,19 @@
+---
+id: installation
+title: Installing BigTest
+slug: /
+---
+
+The `bigtest` package includes everything you need to get started sogo ahead and install BigTest to your project:
+```
+$ yarn add -D bigtest
+```
+
+Once that's finished installing, you can run the following command:
```

Comment:
Where should I run this? is it the within the project that I want to test or is it arbitrary?

Reply:
Both commands can be run from anywhere in the project. I think it does a thing where it searches up the directory until it hits package.json I assume because you can run `yarn bigtest init` from `./test/` or even `./node_modules` and it'll create/modify `bigtest.json` at the root of the project.

I'll need to confirm how it works in a monorepo.